Docs/user-guides/reference/02-working-with-openl-tables/03-table-types/02-decision-table/11-external-tables-usage-in-smart-decision-tables.md

```
@@ -0,0 +1,31 @@
+##### External Tables Usage in Smart Decision Tables
+
+Conditions, returns, and actions declarations can be separated and stored in specific tables and then used in Smart Decision Tables via column titles. It allows using the Smart Table type for Decision rule even in case of the complicated condition or return calculation logic. Another benefit is that condition and return declarations can be reused in several rules, for example, Conditions table as a template. An example is as follows.
+
+![](../../../../ref_guide_images/7db77e6166ea613d591968acde60c104.png)
+
+*Using external conditions in a smart rules table*
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)